test/plug_signature/signature_string_test.exs

```diff
@@ -255,7 +255,14 @@ defmodule PlugSignature.SignatureStringTest do
         [path, query] -> {path, query}
       end
 
+    host =
+      case List.keyfind(headers, "host", 0) do
+        {"host", host} -> host
+        nil -> nil
+      end
+
     %Plug.Conn{
+      host: host || "www.example.com",
```

Comment:
`Plug.Conn` sets the `host` to `www.example.com` by default since forever.
But now, we need to do it explicitly since we are providing the host.